tests/README.md

```diff
@@ -86,6 +86,18 @@ tmt run -v -eWITH_VALGRIND=1
 
 If `valgrind` detects a memory leak in a test, the test will fail, and the logs will be found in the test `data` directory.
 
+## Running integration tests with code coverage report
+
+To produce code coverage report during integration tests run set WITH_COVERAGE environment variable as follows:
```

Comment:
Could you please either  mention that RPMs need to be built with coverage support to be able to run integration test with coverage report or move section about build RPMs with coverage support into this section